### PR TITLE
adds more cursed animal masks

### LINKED ISSUE #53519: horse masks don't impede speech, but pig masks do
from the curse of the barnyard spell

image

### PR DESCRIPTION
**Pull request by ShizCalev**
Standardized how cursed effects are done. they had a bunch of copypasta things between one another, so figured might as well just make it one system.
Allows admins to varedit masks to make them cursed, remove their curse, ect.

:cl: ShizCalev
fix: Fixed a lot of animal masks not counting as small objects.
fix: Fixed horse / pig / cow masks not being counter as face coverings in mask checks.
fix: Fixed a consistency issue where picking up some cursed masks and equipping it wasn't sending the playing a message that the mask was cursed.
fix: Fixed a minor inconsistency between the descriptions of cursed masks.
fix: Fixed cow masks not fully covering a player's head in certain directions.
admin: Admins are now able to varedit an animal masks' cursed var to change if it's cursed or not!
admin: Admins can now change the phrases that cursed masks say.
admin: Admins can now enable and disable a mask's speech effects by editing it's modifies_speech variable.
add: Added more animal masks to the pool for wizard mask spells!
/:cl:

closes #53519

### CODE REVIEW

File: code/modules/clothing/masks/_masks.dm [hunk 1]
```diff
@@ -26,6 +26,17 @@
...
+		var/mob/M = loc
+		if(M.get_item_by_slot(ITEM_SLOT_MASK) == src)
+			if(vval)
+				if(!modifies_speech)
+					RegisterSignal(M, COMSIG_MOB_SAY, .proc/handle_speech)
+			else if(modifies_speech)
+				UnregisterSignal(M, COMSIG_MOB_SAY)
+	return ..()
+
 /obj/item/clothing/mask/proc/handle_speech()
```
Comment (Jared-Fogle) [inline code/modules/clothing/masks/_masks.dm:40]: If this is being used as a signal handler now, it needs to have `SIGNAL_HANDLER` on it.

Comment (Jared-Fogle) [reply code/modules/clothing/masks/_masks.dm:40]: Actually, it appears that this being a signal handler isn't new. Can you add it anyway if it doesn't break anything?

Comment (ShizCalev) [reply code/modules/clothing/masks/_masks.dm:40]: a ton of handle_speech and handle_hearing procs are missing it, i'll leave that for another PR though

**Review by Twaticus — APPROVED** [review]

**Comment by ShizCalev** [comment]
now tested. just need to fix merge conflict and this is good to merge.

**Comment by ShizCalev** [comment]
Ready

**Pull request merged**